data/projects.json

```
@@ -248,5 +248,15 @@
         "level": "basic",
         "site": "",
         "tags": "score-Keeper, HTML, CSS, JAVASCRIPT"
+    },   
+    {
+        "name": "FNDetectoApp",
+        "display": "FNDetectoApp",
+        "description": "FNDetectoApp is a simple fake news detection web app which detects fake news and real news .",
```

The description could be more descriptive and do not use the name, see my suggestion.
```suggestion
        "description": "Fake news detection web app which detects real or fake news using the machine learning.",
```